staging/src/k8s.io/legacy-cloud-providers/gce/gce_instances.go

```diff
@@ -424,54 +425,68 @@ func (g *Cloud) AddAliasToInstance(nodeName types.NodeName, alias *net.IPNet) er
 }
 
 // Gets the named instances, returning cloudprovider.InstanceNotFound if any
-// instance is not found
-func (g *Cloud) getInstancesByNames(names []string) ([]*gceInstance, error) {
+// instance is not found. Always returns the instances that could be found.
+// Maximum names per request is set to the default page size of 500.
+func (g *Cloud) getInstancesByNames(allNames []string) ([]*gceInstance, error) {
```

Comment:
You still didn't change the interface and I definitely agree that partial-result+error is a bad thing to use.

Can you please open a separate PR and do whatever changes are needed for this function for the purpose of this PR there?

Reply:
I just opened https://github.com/kubernetes/kubernetes/pull/80180. I'll rebase this when that is merged.